scripts/seed/datasets/integrated/generate.py

```
@@ -0,0 +1,320 @@
+            artist_id=random.randint(1,artist_count); album_id=random.randint(1,album_count); track_number=random.randint(1,20); featured_artist_text=None if random.random()<0.8 else f"Featured Artist {random.randint(1,1000)}"
+        else:
+            artist_id=album_id=track_number=featured_artist_text=None
+        created_at=random_datetime_between(BASE_DATE,NOW); updated_at=random_datetime_between(created_at,NOW)
+        published_at=random_datetime_between(created_at,NOW) if status=="PUBLISHED" else None
+        scheduled_publish_at=None if status=="PUBLISHED" else created_at+timedelta(days=random.randint(1,7))
+        yield [track_id, owner_user_id, track_type, nullable(artist_id), nullable(album_id), nullable(track_number), f"Track {track_id}", f"Lyrics for track {track_id}", f"GENRE_{random.randint(1,20)}", f"https://cdn.fivefy.local/audio/{track_id}.mp3", random.randint(60,420), nullable(featured_artist_text), status, format_dt(scheduled_publish_at), format_dt(published_at), int(random.paretovariate(1.5)*100), format_dt(created_at), format_dt(updated_at), format_dt(random_datetime_between(updated_at,NOW) if random.random()<0.05 else None)]
+
+def generate_likes(count: int, user_count: int, track_count: int):
+    used = set()
+    for like_id in range(1, count + 1):
+        user_id = random.randint(1, user_count)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)